code/src/components/GameBoard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { MainGame } from './MainGame';
+import { StartPage } from './StartPage';
+import { useSelector } from 'react-redux';
+
+const GameBoard = ({ startBgMusic }) => {
+  const game = useSelector((state) => state.game);
+  return (
```

Comment:
I really like your gameboard component. It's very clean like app.js but very functional, without cluttering up confusing another component